scanapi/reporter.py

```diff
@@ -29,7 +29,8 @@ def write(self, results):
         logger.info("\nThe documentation was generated successfully.")
         logger.info(f"It is available at {self.output_path}")
 
-    def _build_context(self, results):
+    @staticmethod
+    def _build_context(results):
```

Comment:
Sorry, I did not get the idea of changing this method to a static one. Would you mind to give me some context?
And I believe we would need to change the related calls too.